src/main/java/org/opentripplanner/middleware/otp/response/Step.java

```diff
@@ -2,15 +2,14 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonInclude;
-
-import java.util.Objects;
+import org.opentripplanner.middleware.utils.Coordinates;
 
 /**
  * Plan response, step information. Produced using http://www.jsonschema2pojo.org/
  */
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class Step implements Cloneable {
+public class Step extends Coordinates implements Cloneable {
```

Comment:
@br648 Making `Step` and `Place` extend `Coordinates` was maybe a step too far, but resulted in code simplication.

Reply:
JSON deserialization from OTP response still works, but that broke Swagger and other things, so now I'm thinking doing an interface with a `toCoordinates` method.